services/queue/src/api.js

```diff
@@ -1949,7 +1949,7 @@ builder.declare({
   },
   name: 'listWorkers',
   scopes: 'queue:list-workers:<provisionerId>/<workerType>',
-  stability: APIBuilder.stability.experimental,
+  stability: APIBuilder.stability.deprecated,
```

Comment:
Hmm, why do you want to deprecate `listWorkers` and `getWorker`? Afaik they are quite popular and are used outside if TC UI (cli tools).

Reply:
Actually yes, it was marked as experimental anyway, but we should keep it for at least 1 major release more I think